src/PeriodEnd/TransferPayments/SFA.DAS.ProviderPayments.Calc.TransferPayments/Dal/Repositories/TransferRepository.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using SFA.DAS.Payments.DCFS.Infrastructure.Data;
+using SFA.DAS.ProviderPayments.Calc.TransferPayments.Dal.DatabaseEntities;
+using SFA.DAS.ProviderPayments.Calc.TransferPayments.Dependencies;
+using SFA.DAS.ProviderPayments.Calc.TransferPayments.Dto;
+
+namespace SFA.DAS.ProviderPayments.Calc.TransferPayments.Dal.Repositories
+{
+    class TransferRepository : DcfsRepository, IAmATransferRepository
+    {
+        public TransferRepository(string connectionString) : base(connectionString)
+        {
+        }
+
+        public void SaveTransfers(List<TransferPaymentSet> transfers)
+        {
+            ExecuteBatch(transfers.SelectMany(x => x.TransferPayments).ToArray(), "RequiredTransferPayments.Payments");
+            ExecuteBatch(transfers.SelectMany(x => x.AccountTransfers).ToArray(), "TransferPayments.AccountTransfers");
+        }
+
+        public IEnumerable<RequiredTransferPayment> RequiredTransferPayments()
+        {
+            var command = "SELECT RequiredPaymentId, " +
```

Comment:
Not a fan of concatenated strings. Can this be made into a literal like some of the other components?

Reply:
could be a constant